hortonworks/cloudbreak
# CB-20250 Support the "Cancelled" datalake backup state

# Motivation
Currently cancelled datalake backups are shown as "successful" in Cloudera Management Console. We want to indicate cancelled backups as "Cancelled" and also send a corresponding audit event

# Test approach
- create an environment
- start a DL backup (cli) and wait for it to complete. 
- verify the backup status on CMC is shown as "successful"
- verify an event log on CMC is having an event "successful backup"
- start a DL backup (cli) and cancel it (cli) at any state.
- verify the backup status on CMC is shown as "cancelled"
- verify an event log on CMC is having an event "backup cancelled"
